app/lib/audit/checksum.rb

```diff
@@ -10,9 +10,9 @@ def self.validate_disk(endpoint_name, limit=Settings.c2m_sql_limit)
       # pcs_w_expired_fixity_check is an AR Relation; it could return a lot of results, so we want to process it in
```

Comment:
whoops - looks like the comment wasn't updated with the var name chagne.  Also not sure why var name change was necessary, but whatever.